cmd/mesh/manifest.go

```diff
@@ -28,7 +28,8 @@ import (
 	"istio.io/operator/pkg/translate"
 	"istio.io/operator/pkg/util"
 	"istio.io/operator/pkg/validate"
-	"istio.io/operator/pkg/version"
+
+	binversion "istio.io/operator/version"
```

Comment:
Why is this alias needed? Also, there's an extra space between `validate` and `version` packages.

Reply:
I added this for consistency - we now have two version packages: a library under pkg/version and the compiled in version in version/version (which we could rename I suppose).